rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -266,6 +266,22 @@ NodeGraph::count_subscribers(const std::string & topic_name) const
   return count;
 }
 
+std::vector<rmw_topic_info_t>
```

Comment:
`rmw_topic_info_t` is too "C style", specially because of the C style strings.
I think that the way to go is creating an equivalent `cpp` class.
It will force some extra unnecessary copying, but performance is not much important in graph instronspection functions.